<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisitEditView</class>
 <widget class="QWidget" name="VisitEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>就诊记录</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="2">
      <widget class="QLineEdit" name="dbEditMedicationList"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="dbDname">
       <property name="text">
        <string>医生姓名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="dbVisitID">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="dbEditDiagnosis"/>
     </item>
     <item row="3" column="2">
      <widget class="QDateEdit" name="dbEditDate">
       <property name="styleSheet">
        <string notr="true"> border: 2px solid gray;
      border-radius: 10px;
      padding: 0 8px;</string>
       </property>
       <property name="calendarPopup">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="dbDiagnosis">
       <property name="text">
        <string>诊断结果</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="dbPname">
       <property name="text">
        <string>病人姓名:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="dbEditID"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="dbDate">
       <property name="text">
        <string>就诊日期:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="dbEditPname"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="dbMedicationList">
       <property name="text">
        <string>开具药品</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="dbEditDname"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>83</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>84</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
